backend/setupWebSession.js

```diff
@@ -13,14 +13,14 @@ module.exports = () => {
     const { enabled, host, port, password } = config.redis
     if (!enabled || !host) return null
 
-    const client = redis.createClient({
```

Comment:
Nots ure why you bothered to change this - using the `redis` object is how it works in our templates.

Not that important - just wondering why it was changed.

Reply:
I just thought it looked tidier - I made the same changes in the other repositories when I updated

Reply:
ah ok